projects/plugins/jetpack/unauth-file-upload.php

```diff
@@ -69,10 +69,15 @@ function handle_file_download() {
 	 */
 	$file = apply_filters( 'jetpack_unauth_file_upload_get_file', array(), $file_id );
 
-	if ( is_wp_error( $file ) || empty( $file ) ) {
+	if ( is_wp_error( $file ) || empty( $file ) || ! is_array( $file ) ) {
 		wp_die( esc_html__( 'Error retrieving file content.', 'jetpack' ) );
 	}
 
+	// Given $file can be manipulated by a filter, make sure everything is as it should be.
+	$file['content'] = $file['content'] ?? '';
+	$file['type']    = $file['type'] ?? 'application/octet-stream';
+	$file['name']    = $file['name'] ?? '';
```

Comment:
Why not use the [Null Coalescing Assignment Operator](https://wiki.php.net/rfc/null_coalesce_equal_operator)?

```suggestion
	$file['content'] ??= '';
	$file['type']    ??= 'application/octet-stream';
	$file['name']    ??= '';
```

Reply:
That doesn't exist until PHP 7.4, and we still have to support back to 7.2. Eventually!